src/app/theming/animations/animations-sample-1/animations-sample-1.component.scss

```diff
@@ -0,0 +1,20 @@
+.sample-wrapper {
```

Comment:
You can target the `:host` selector to achieve the same, without needing an extra container.